parl/algorithm_zoo/simple_algorithms.py

```diff
@@ -0,0 +1,153 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from parl.framework.net import Algorithm
+import parl.layers as layers
+from parl.layers import common_functions as comf
+
+
+class SimpleAC(Algorithm):
+    """
+    A simple Actor-Critic that has a feedforward policy network and
+    a single discrete action.
+    """
+
+    def __init__(self,
+                 model_func,
+                 num_actions,
+                 mlp_layer_confs,
```

Comment:
why do we need mlp_layer_conf? Why not to make it part of Model?

Reply:
This is just an example class for Algorithm. The user can assume that there are additional parameters in an algorithm, or they can assume that all the parameters are in the Model. Either way, we can guarantee that the parameters are correctly handled when syncing paras from one algorithm to another.